### ISSUE TITLE
Hyperparameter Optimization with Keras for the Iris Prediction.ipynb error

### ISSUE THREAD

**Issue by mtulunay** [autonomio/talos#185]
- [X ] I'm up-to-date with the latest release:
          
- I'm using Talos example notebook (Hyperparameter Optimization with Keras for the Iris Prediction.ipynb on Windows 10
A Very Short Introduction to Hyperparameter Optimization of Keras Models with Talos sample is worked

Error cell: 
h = ta.Scan(x, y, params=p,
            model=iris_model,
            dataset_name='iris',
            experiment_no='1',
            grid_downsample=.01)

---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
C:\ProgramData\Anaconda3\lib\site-packages\talos\scan\scan_round.py in scan_round(self)
     31     try:
---> 32         _hr_out, self.keras_model = ingest_model(self)
     33     except TypeError as err:

C:\ProgramData\Anaconda3\lib\site-packages\talos\model\ingest_model.py in ingest_model(self)
      9                       self.y_val,
---> 10                       self.round_params)

<ipython-input-4-125f26e8ea75> in iris_model(x_train, y_train, x_val, y_val, params)
     23                     validation_data=[x_val, y_val],
---> 24                     callbacks=early_stopper(params['epochs'], mode='strict'))
     25 

C:\ProgramData\Anaconda3\lib\site-packages\keras\engine\training.py in fit(self, x, y, batch_size, epochs, verbose, callbacks, validation_split, validation_data, shuffle, class_weight, sample_weight, initial_epoch, steps_per_epoch, validation_steps, **kwargs)
   1038                                         steps_per_epoch=steps_per_epoch,
-> 1039                                         validation_steps=validation_steps)
   1040 

C:\ProgramData\Anaconda3\lib\site-packages\keras\engine\training_arrays.py in fit_loop(model, f, ins, out_labels, batch_size, epochs, verbose, callbacks, val_f, val_ins, shuffle, callback_metrics, initial_epoch, steps_per_epoch, validation_steps)
    105                 stateful_metrics=model.stateful_metric_names))
--> 106     _callbacks += (callbacks or []) + [model.history]
    107     callbacks = cbks.CallbackList(_callbacks)

TypeError: unsupported operand type(s) for +: 'EarlyStopping' and 'list'

During handling of the above exception, another exception occurred:

TalosReturnError                          Traceback (most recent call last)
<ipython-input-6-c16983be523d> in <module>
      3             dataset_name='iris',
      4             experiment_no='1',
----> 5             grid_downsample=.01)

C:\ProgramData\Anaconda3\lib\site-packages\talos\scan\Scan.py in __init__(self, x, y, params, model, dataset_name, experiment_no, x_val, y_val, val_split, shuffle, round_limit, grid_downsample, random_method, seed, search_method, reduction_method, reduction_interval, reduction_window, reduction_threshold, reduction_metric, reduce_loss, last_epoch_value, clear_tf_session, disable_progress_bar, print_params, debug)
    161         # input parameters section ends
    162 
--> 163         self._null = self.runtime()
    164 
    165     def runtime(self):

C:\ProgramData\Anaconda3\lib\site-packages\talos\scan\Scan.py in runtime(self)
    166 
    167         self = scan_prepare(self)
--> 168         self = scan_run(self)

C:\ProgramData\Anaconda3\lib\site-packages\talos\scan\scan_run.py in scan_run(self)
     18                      disable=self.disable_progress_bar)
     19     while len(self.param_log) != 0:
---> 20         self = scan_round(self)
     21         self.pbar.update(1)
     22     self.pbar.close()

C:\ProgramData\Anaconda3\lib\site-packages\talos\scan\scan_round.py in scan_round(self)
     35             raise TalosTypeError("Activation should be as object and not string in params")
     36         else:
---> 37             raise TalosReturnError("Make sure that input model returns 'out, model' where out is history object from model.fit()")
     38 
     39     # set end time and log

TalosReturnError: Make sure that input model returns 'out, model' where out is history object from model.fit()

**Comment by mikkokotila** [comment]
Thanks. There was an issue with the example since some things have been streamlined since the notebook was made. I've just posted a fixed version [here](https://github.com/autonomio/talos/blob/daily-dev/examples/Hyperparameter%20Optimization%20with%20Keras%20for%20the%20Iris%20Prediction.ipynb). Note that it's only available in the daily-dev branch for now, but the link takes you to it directly. 

Thanks again. Closing here. Feel free to open new issue/s in case anything.
